pkg/controller/deployable/deployable_utils.go

```diff
@@ -81,6 +81,16 @@ func SyncDeployable(metaobj *unstructured.Unstructured, explorer *utils.Explorer
 func updateDeployableAndObject(dpl *dplv1.Deployable, metaobj *unstructured.Unstructured,
 	explorer *utils.Explorer, hubSynchronizer synchronizer.HubSynchronizerInterface) error {
 
+	// Find the root resource with no owner reference
+	rootobj, err := findRootResource(metaobj, explorer)
+	if err == nil {
+		// Replace metaobj with rootobj
+		metaobj = rootobj
+	} else {
+		klog.Error("Failed to locate parent resource for deployable ", dpl.Namespace+"/"+dpl.Name)
```

Comment:
can we use the resource name instead of the deployable name? The deployable is an object created on the hub , I think the log should contain the name of the actual resource on the managed cluster